.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.12"]
+        python-version: ["3.9", "3.10", "3.12"]
```

Comment:
Let's add Python v3.11 to the list, too.

```yaml
python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Reply:
This change was made in the following commit